pkg/server/endpoints/endpoints.go

```diff
@@ -187,9 +190,10 @@ func (e *endpoints) runHTTPServer(ctx context.Context, server *http.Server) erro
 		return err
 	case <-ctx.Done():
 		e.c.Log.Info("Stopping HTTP server")
-		server.Close()
 		l.Close()
+		server.Close()
```

Comment:
Curious about this too w.r.t. above comment

Reply:
see previous comment